### DISCUSSION TITLE
halo如何将文章或者读取git仓库上面的笔记呢？

### DISCUSSION THREAD

**Discussion by txbxxx** [halo-dev/halo#7146] [Q&A]
我想讲笔记存储至github或者其他仓库上，请问halo如何读取呀

**Comment by JohnNiang** [comment]
Hi @txbxxx , thank you for reaching out here!

目前暂时还没有这个功能。

**Reply by txbxxx** [reply]
> Hi @txbxxx , thank you for reaching out here!
> 
> 目前暂时还没有这个功能。

谢谢大佬
